src/project/qml/MuseScore/Project/AskSaveLocationTypeDialog.qml

```diff
@@ -72,6 +72,9 @@ You can share drafts with others and publish your finished scores publicly too."
 
                 imageSource: "internal/SaveToCloud/images/Cloud.png"
 
+                navigationPanel.section: root.navigationSection
+                navigationPanel.order: 1
```

Comment:
I think, 'Save to the cloud' and 'Save to computer' should be inside the same panel for navigating with arrow keys instead of Tab